compiler/rustc_error_codes/src/error_codes/E0756.md

```diff
@@ -21,6 +21,9 @@ extern "C" {
     #[ffi_const] // ok!
     pub fn strlen(s: *const i8) -> i32;
 }
+
+#[ffi_pure]
+extern "C" pub fn foo() {} // ok!
```

Comment:
I don't see what it's adding. :-/

Reply:
An alternative `extern` syntax I mentioned last time. I didn't know about `extern "C" { ... }` before this but I only know `extern "C" ...`, one thing is to let others know that extern could be not a block.

Reply:
Weird, it doesn't work. I thought it works.